internal/pkg/service/stream/api/mapper/source_response.go

```diff
@@ -68,6 +68,28 @@ func (m *Mapper) NewSourcesResponse(
 	}, nil
 }
 
+func (m *Mapper) NewSourceVersions(
+	ctx context.Context,
+	afterId string,
+	limit int,
+	list func(...iterator.Option) iterator.DefinitionT[definition.Source],
+) (*api.EntityVersions, error) {
+	sources, page, err := loadPage(ctx, afterId, limit, etcd.SortAscend, list, m.NewSourceResponse)
```

Comment:
Is `NewSourceResponse` correct here? :thinking: I'd expect `NewVersionResponse`.

Reply:
Yeah I think the loop over sources below isn't necessary if you fix this.

Reply:
Ah okay nevermind it's because ListVersions returns ` iterator.DefinitionT[definition.Source]` so it's probably intentional like this. Alright then.